### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by blemia51**

### CODE REVIEW

**Comment by rocambille** [comment]
there are conflicts to resolve in Converter.js :/

**Comment by blemia51** [comment]
conflicts resolved

File: yodaconvert/src/components/Converter.js [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+        }
+        this.handleInputChange = this.handleInputChange.bind(this)
     }
+
+
+handleInputChange(e) {
+    this.setState({ 
+        inputText: e.target.value,
+        outputText : convertToHTML(this.state.inputText) })
+      }
```
Comment (rocambille) [inline yodaconvert/src/components/Converter.js]: fix indentation, and avoid reading the state while writing it:

      handleInputChange(e) {
        const newText = e.target.value
        this.setState({ 
          inputText: newText,
          outputText : convertToHTML(newText)
        })
      }

File: yodaconvert/src/components/Converter.js [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
+    console.log(this.state)
     return (
         <div className="textContainer">
-            <textarea className="input-text" rows="6" cols="40"></textarea>
+            
+            <textarea className="input-text" name="inputText" rows="6" cols="40" value={this.state.inputText} onChange={this.handleInputChange}>    
+            </textarea>
             <ButtonConvDel/>
-            <textarea className="output-text" rows="6" cols="40"></textarea>
+            <textarea className="output-text" name="outputText" rows="6" cols="40" value={this.state.outputText} onChange={this.handleInputChange} >
```
Comment (rocambille) [inline yodaconvert/src/components/Converter.js]: shouldn't this one be read only?

see https://www.w3schools.com/tags/att_textarea_readonly.asp

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**